weed/pb/volume_server.proto

```diff
@@ -147,6 +147,7 @@ message VacuumVolumeCompactRequest {
 }
 message VacuumVolumeCompactResponse {
     int64 processed_bytes = 1;
+    float percent_load_avg_1m = 2;
```

Comment:
float is not valid. Need to use float32

Reply:
change percent_load_avg_1m to load_avg_1m

Reply:
> float is not valid. Need to use float32

protoc thinks differently
![Screenshot 2022-08-01 at 21 25 44](https://user-images.githubusercontent.com/9497591/182197735-38c49685-f6d9-4ad3-98dc-dbf1346389c5.png)

Reply:
> change percent_load_avg_1m to load_avg_1m

ok